<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>excelTransLineDialog</class>
 <widget class="QDialog" name="excelTransLineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excel Export Parameter Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <widget class="CardWidget" name="CardWidget_5">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="BodyLabel" name="BodyLabel_3">
        <property name="text">
         <string>Select Excel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LineEdit" name="excelLE">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ToolButton" name="selectExcelFilePB"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="BodyLabel" name="BodyLabel_2">
        <property name="text">
         <string>Select Line Name Field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ComboBox" name="lineNameFieldCb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="BodyLabel" name="BodyLabel_5">
        <property name="text">
         <string>Select Point Id Field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ComboBox" name="pointIdFieldCb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_4">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="BodyLabel" name="BodyLabel_6">
        <property name="text">
         <string>Select X Field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ComboBox" name="XFieldCb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget_6">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="BodyLabel" name="BodyLabel_7">
        <property name="text">
         <string>Select Y Field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ComboBox" name="YFieldCb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="CardWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="BodyLabel" name="BodyLabel_4">
        <property name="text">
         <string>Result Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LineEdit" name="resLE">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ToolButton" name="selectSaveFilePB"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PushButton" name="runPB">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="cancelPB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../yoyirs.qrc"/>
 </resources>
 <connections/>
</ui>
